TrailBlaze/Networking/Interceptor.m

```diff
@@ -119,7 +119,8 @@ + (void) getBestETAPoint: (int) maxWaitTime allPoints:(NSArray *) allPoints inte
         if ([bestDifference floatValue] <= etaDifferenceThreshold) {
         completion([etaPointPairs objectForKey:[NSString stringWithFormat:@"%@", bestDifference]], nil);
         } else {
-            completion(nil, nil);
+            NSError *err;
+            completion(nil, err);
```

Comment:
re-iterating my prev comment: let's pass back a real error value with details on why the caller isn't getting an ETA.

Reply:
I rewrote it to use either error from the first or second of the nested eta requests depending of which of those errors out